src/wix/WixToolset.Core/Compiler_Package.cs

```diff
@@ -2521,7 +2523,8 @@ private void ParseSequenceElement(XElement node, SequenceTable sequenceTable)
                         case "Dialog":
                             if (showDialog)
                             {
-                                actionName = this.Core.GetAttributeIdentifierValue(childSourceLineNumbers, attrib);
+                                actionIdentifier = this.Core.GetAttributeIdentifier(childSourceLineNumbers, attrib);
+                                actionName = actionIdentifier.Id; //this.Core.GetAttributeIdentifierValue(childSourceLineNumbers, attrib);
```

Comment:
nit: remove comment

Reply:
Fixed locally. I will push a rebase at the end of your review @barnson.